### PR TITLE
Add issues visualisation

### PR DESCRIPTION
**Pull request by KarlPatach**

### CODE REVIEW

File: force-app/main/default/objects/GitHub_Pull_Requests__x/GitHub_Pull_Requests__x.object-meta.xml [hunk 1]
```diff
@@ -117,8 +117,8 @@
     <description>GitHub Pull Requests</description>
     <enableFeeds>false</enableFeeds>
     <enableLicensing>false</enableLicensing>
-    <enableReports>true</enableReports>
-    <enableSearch>true</enableSearch>
+    <enableReports>false</enableReports>
```
Comment (juanka588) [inline force-app/main/default/objects/GitHub_Pull_Requests__x/GitHub_Pull_Requests__x.object-meta.xml:120]: true

File: force-app/main/default/objects/GitHub_Repos__x/GitHub_Repos__x.object-meta.xml [hunk 2]
```diff
@@ -118,7 +118,7 @@
     <enableFeeds>false</enableFeeds>
     <enableLicensing>false</enableLicensing>
     <enableReports>true</enableReports>
-    <enableSearch>true</enableSearch>
+    <enableSearch>false</enableSearch>
```
Comment (juanka588) [inline force-app/main/default/objects/GitHub_Repos__x/GitHub_Repos__x.object-meta.xml:121]: revert

File: force-app/main/default/objects/GitHub_Pull_Requests__x/GitHub_Pull_Requests__x.object-meta.xml (hunk 1 above)
Comment (KarlPatach) [reply force-app/main/default/objects/GitHub_Pull_Requests__x/GitHub_Pull_Requests__x.object-meta.xml]: Done

File: force-app/main/default/objects/GitHub_Repos__x/GitHub_Repos__x.object-meta.xml (hunk 2 above)
Comment (KarlPatach) [reply force-app/main/default/objects/GitHub_Repos__x/GitHub_Repos__x.object-meta.xml:121]: Done

**Pull request merged**